### ISSUE TITLE
Render waterways with seasonal=yes the same way as intermittent=yes

### ISSUE THREAD

**Issue by liotier** [gravitystorm/openstreetmap-carto#2095]
Seasonality is a specific sort of intermittency - variation is according to seasons instead of more randomly weather-related.

For now, seasonal waterways appear as solid lines and it is a bit disconcerting.

Treating seasonal=yes the same way as intermittent=yes feels quite correct to me and it would also minimize the work necessary to take this extra tag into account.

**Comment by matkoniecz** [comment]
Is it necessary for correctly tagged objects?

https://wiki.openstreetmap.org/wiki/Key:intermittent

> The intermittent=* key is used to indicate that a body of water does not permanently contain water but dries up some times - either regularly depending on seasons or based on other conditions. 

So seasonal would be tagged with both seasonal key and intermittent=yes

**Comment by matkoniecz** [comment]
Is there any case where seasonal would be used (except seasonal=no) where intermittent=yes is not supposed to be used?

**Comment by kocio-pl** [comment]
As far as I understand it, seasonal is a special case of intermittent: we know it has some time pattern. I would not add intermittent for seasonal, since it's implied and we don't add default tags (just like we woudn't add car=yes for a highway=motorway). Maybe wiki should be updated to make it clear.

What do you think about it?

**Comment by liotier** [comment]
On Wed, 10 Jan 2018 08:20:08 -0800
kocio-pl <notifications@github.com> wrote:
>
> As far as I understand it, seasonal is a special case of
> intermittent: we know it has some time pattern. I would not add
> intermittent for seasonal, since it's implied and we don't add
> default tags (just like we woudn't add car=yes for a
> highway=motorway). Maybe wiki should be updated to make it clear.
> 
> What do you think about it?

That is how I understand it too.

**Comment by matkoniecz** [comment]
I would consider tagging only seasonal without intermittent like tagging only "parking=surface", without amenity=parking (additional specific detail without more generic tag).

**Comment by kocio-pl** [comment]
I think current documentation supports our point of view - `key:parking` wiki page has "requires amenity=parking":

https://wiki.openstreetmap.org/wiki/Key:parking

and seasonal has only "see also intermittent".

This way or another wiki should be updated to make it clear if intermittent is required (and should be always tagged along) or implied (and should be never used with seasonal), so probably Tagging list discussion is needed.

**Comment by matkoniecz** [comment]
> This way or another wiki should be updated to make it clear if intermittent is required (and should be always tagged along) or implied (and should be never used with seasonal), so probably Tagging list discussion is needed.

I agree, I created https://wiki.openstreetmap.org/wiki/Key:seasonal#Relation_to_intermittent (attempted to describe both sides) and will post to tagging mailing list

**Comment by matkoniecz** [comment]
Posted to mailing list, lets discuss there.

**Comment by kocio-pl** [comment]
What was the output of the discussion?

**Comment by matkoniecz** [comment]
https://lists.openstreetmap.org/pipermail/tagging/2018-January/034688.html was the only relevant reply

**Comment by kocio-pl** [comment]
Unfortunately I have trouble understanding what's the position of @imagico with this after reading second and third time. Could you summarize it?

**Comment by matkoniecz** [comment]
Seasonal without intermittent is considered as good tagging by him.

Generally, it seems that my opinion

> I would consider tagging only seasonal without intermittent like tagging only "parking=surface", without amenity=parking (additional specific detail without more generic tag).

is not shared by others.
